javacord-api/src/main/java/org/javacord/api/DiscordApiBuilder.java

```diff
@@ -297,6 +298,18 @@ public boolean isWaitingForServersOnStartup() {
         return delegate.isWaitingForServersOnStartup();
     }
 
+    /**
+     * WIP: to work properly you have to use at least {@link IntentType#GUILD_PRESENCES} and {@link IntentType#GUILDS}
+     * Sets intent for the events which should be received.
+     *
+     * @param intentType One or more intents from {@link IntentType}.
+     * @return The current instance in order to chain call methods.
+     */
+    public DiscordApiBuilder setIntent(IntentType... intentType) {
```

Comment:
Please also update all other setters accordingly.
```suggestion
    public DiscordApiBuilder setIntents(Intent... intent) {
```